saleor/graphql/channel/types.py

```diff
@@ -324,7 +324,7 @@ class Channel(ModelObjectType):
     class Meta:
         description = "Represents channel."
         model = models.Channel
-        interfaces = [graphene.relay.Node]
+        interfaces = [graphene.relay.Node, ObjectWithMetadata]
```

Comment:
Can you add here also:
```python
        metadata_since = ADDED_IN_315
```
(This will fix the schema description. Currently, it shows that the schema was added in 3.3.)